src/ckb/gen/cfn.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::all)]
```

Comment:
https://github.com/nervosnetwork/cfn-node/blob/d1cdb4f2464beab34e962e6b1b8aeac85b40d74b/src/ckb/gen/mod.rs#L2 does not work?

Reply:
yes, it works to add another `#[allow(clippy::all)]`.